File: src/main/java/com/sm/carwashmonitor/dto/StatisticsHighlightsDTO.java
package com.sm.carwashmonitor.dto;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@RequiredArgsConstructor
@Getter
@Setter
public class StatisticsHighlightsDTO {
    Float totalRevenue;
    String bestStationName;
    Float bestStationRevenue;
    Integer totalWashCount;
    Float totalWaterConsumption;
    Float totalWaxConsumption;
    Float totalDetergentConsumption;
    Float totalWaterCost;
    Float totalWaxCost;
    Float totalDetergentCost;
}

File: Dtos/StatisticsHighlightsDto.cs
namespace CarwashMonitor.Dtos
{
    public class StatisticsHighlightsDto
    {
        public float TotalRevenue { get; set; }
        public string? BestStationName { get; set; }
        public float BestStationRevenue { get; set; }
        public int TotalWashCount { get; set; }
        public float TotalWaterConsumption { get; set; }
        public float TotalWaxConsumption { get; set; }
        public float TotalDetergentConsumption { get; set; }
        public float TotalWaterCost { get; set; }
        public float TotalWaxCost { get; set; }
        public float TotalDetergentCost { get; set; }
    }
}
